ui/src/components/Organization/List.tsx

```diff
@@ -9,17 +9,33 @@ interface Props {
   organizations: Organization[];
 }
 
+const defaultBand = (brand: string, name: string) => {
+  if (brand && brand.length) {
+    return <img className={c('brand')} src={brand} />;
+  } else {
+    const firstLetter = name.split('')[0];
+    return (
+      <div className={c('default-brand')}>
+        <span className={c('default-icon')}>{firstLetter}</span>
+      </div>
+    );
+  }
+};
+
 const OrgItem: React.FC<Organization> = (p) => (
-  <div className={c('content')} key={p.id}>
-    <div className={c('section1')}>
-      <img className={c('logo')} src={p.square_logo} />
-      <span className={c('name')}>{p.name}</span>
+  <div className={c('container')}>
```

Comment:
I was thinking we are unnecessarily paying extra price of a function call for every className. How do you feel about setting a new convention: 
1. Import styles as `c` (for classes)
2. Create a binding `cx` for classnames library
3. Use `c["class-name"] unless we need conditional classes, in which case we use `cx`

No need to implement it in this PR, just wanted to know your opinion about this.